composer.json

```diff
@@ -200,7 +200,7 @@
         "zaporylie/composer-drupal-optimizations": "1.2.0"
     },
     "require-dev": {
-        "drupal/core-dev": "~10.1.7",
+        "drupal/core-dev": "^10.1",
```

Comment:
We talked about removing this package, but, when I did, all the PHPUnits started to fall apart.